skflow/tests/test_io.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright 2015 Google Inc. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import random
+
+from sklearn import datasets
+from sklearn.metrics import accuracy_score
+
+import tensorflow as tf
+from tensorflow.python.platform import googletest
+
+from skflow.io import *
+import skflow
+
+class BaseTest(googletest.TestCase):
+    def test_pandas_data_classification(self):
+        if HAS_PANDAS:
+            random.seed(42)
+            iris = datasets.load_iris()
+            data = DataFrame(iris.data)
+            labels = DataFrame(iris.target)
```

Comment:
Added test case for this. I originally thought it is automatically covered since Series belongs to DataFrame but it is also safer to do so.